File: 백준/Bronze/18198. Basketball One－on－One/Basketball One－on－One.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main()
{
    char Arr[201];
    int sum_a = 0, sum_b = 0, tmp;

    scanf("%s", &Arr);

    for (int i = 0; i < strlen(Arr); i += 2) {
        if (Arr[i] == 'A') {
            tmp = atoi(&Arr[i + 1]);
            sum_a += tmp;
        }
        else if (Arr[i] == 'B') {
            tmp = atoi(&Arr[i + 1]);
            sum_b += tmp;
        }
    }

    printf("%c", (sum_a > sum_b) ? 'A' : 'B');

    return 0;
}

File: 백준/Bronze/18198. Basketball One－on－One/Basketball One－on－One.py
sum_a = 0
sum_b = 0
Arr = list(input())
for i in range(0,len(Arr),2):
    if Arr[i]=="A":
        sum_a += int(Arr[i+1])
    elif Arr[i]=="B":
        sum_b += int(Arr[i+1])
print('A' if sum_a > sum_b else 'B')